var/spack/repos/builtin/packages/parflow/package.py

```diff
@@ -65,21 +65,18 @@ def setup_run_environment(self, env):
 
     examples_dir = "examples"
 
-    def test(self):
+    def test_smoke(self):
         """Perform smoke test on installed ParFlow package."""
         # Run the single phase flow test
         run_path = join_path(self.spec.prefix, self.examples_dir)
         if os.path.isdir(run_path):
+            options = ["default_single.tcl", "1", "1" "1"]
             with working_dir(run_path):
-                self.run_test(
-                    "{0}/tclsh".format(self.spec["tcl"].prefix.bin),
-                    ["default_single.tcl", "1", "1" "1"],
-                )
+                exe = which("{0}/tclsh".format(self.spec["tcl"].prefix.bin))
+                exe(*options)
```

Comment:
It would be a lot clearer if this method was consistent with the description at line 70 (which is a good docstring for a method called `test_single_phase_flow`).